.gitattributes

```diff
@@ -2,7 +2,7 @@
 *.[ch] whitespace=indent,trail,space diff=cpp
 *.sh whitespace=indent,trail,space eol=lf
 *.perl eol=lf diff=perl
-*.pl eof=lf diff=perl
+*.pl eol=lf diff=perl
```

Comment:
Since this is a bit easy to miss (`f` and `l` look pretty similar in some fonts), it would make a ton of sense to add something like this to the commit message:

> In Git attributes, we clearly are interested in end-of-line characters ("eol"), _not_ in end-of-file markers ("eof").

Reply:
The commit message is modified.